opencontext_py/apps/searcher/new_solrsearcher/configs.py

```diff
@@ -0,0 +1,242 @@
+# function. The tuple is organized as so:
+#
+# 1st element: request GET parameter
+# 2nd element: field to remove from the Solr facet.field list
+#              (usually None).
+# 3rd element: dictionary of key-word arguments to supply to
+#              querymaker.get_general_hierarchic_paths_query_dict 
+# ---------------------------------------------------------------------
+HIERARCHY_PARAM_TO_SOLR = [
+    (
+        'proj', SolrDocument.ROOT_PROJECT_SOLR,
+        {
+            'root_field': SolrDocument.ROOT_PROJECT_SOLR,
+            'field_suffix': SolrDocument.FIELD_SUFFIX_PROJECT,
+        },
+    ),
+    (
+        'prop', None,
+        {
+            'root_field': SolrDocument.ROOT_PREDICATE_SOLR,
+            'field_suffix': SolrDocument.FIELD_SUFFIX_PREDICATE,
+        },
+    ),
+    (
+        'dc-subject', None,
+        {
+            'root_field': 'dc_terms_subject___pred_id',
+            'obj_all_slug': "dc-terms-subject",
+            'field_suffix': SolrDocument.FIELD_SUFFIX_PREDICATE,
+        },
+    ),
+    (
+        'dc-spatial', None,
+        {
+            'root_field': 'dc_terms_spatial___pred_id',
+            'obj_all_slug': 'dc-terms-spatial',
+            'field_suffix': SolrDocument.FIELD_SUFFIX_PREDICATE,
+        },
+    ),
+    (
+        'dc-coverage', None,
+        {
+            'root_field': 'dc_terms_coverage___pred_id',
+            'obj_all_slug': 'dc-terms-coverage',
+            'field_suffix': SolrDocument.FIELD_SUFFIX_PREDICATE,
+        },
+    ),
+    (
+        'dc-temporal', None,
+        {
+            'root_field': 'dc_terms_temporal___pred_id',
+            'obj_all_slug': 'dc-terms-temporal',
+            'field_suffix': SolrDocument.FIELD_SUFFIX_PREDICATE,
+        },
+    ),
+    (
+        'dc-isReferencedBy', None,
+        {
+            'root_field': 'dc_terms_isreferencedby___pred_id',
+            'obj_all_slug': 'dc-terms-isreferencedby',
+            'field_suffix': SolrDocument.FIELD_SUFFIX_PREDICATE,
+        },
+    ),
+    (
+        'dc-creator', None, {
+            'root_field': 'dc_terms_creator___pred_id',
+            'obj_all_slug': 'dc-terms-creator',
+            'field_suffix': SolrDocument.FIELD_SUFFIX_PREDICATE,
+        },
+    ),
+    (
+        'dc-contributor', None,
+        {
+            'root_field': 'dc_terms_contributor___pred_id',
+            'obj_all_slug': 'dc-terms-contributor',
+            'field_suffix': SolrDocument.FIELD_SUFFIX_PREDICATE,
+        },
+    ),
+    (
+        'bibo-status', None,
+        {
+            'root_field': 'bibo_status___pred_id',
+            'obj_all_slug': 'bibo-status',
+            'field_suffix': SolrDocument.FIELD_SUFFIX_PREDICATE,
+        },
+    ),
+]
+
+
+
+
+# ---------------------------------------------------------------------
+# Configs for making solr result sorting
+# ---------------------------------------------------------------------
+
+SOLR_SORT_DEFAULT = 'interest_score desc'
```

Comment:
Is `interest_score` a built in part of Solr?

Reply:
`interest_score` is a field we define in our solr schema for Open Context. It's essentially a score about how "significant" a given record is. The more images, description, etc., the higher the interest score, since it's assumed more interesting things have more description / documentation.